api-spec/definitions/STAC-standalone.yaml

```diff
@@ -1,355 +1,543 @@
+
+        For WGS84 longitude/latitude the values are in most cases the sequence
+        of minimum longitude, minimum latitude, maximum longitude and maximum
+        latitude. However, in cases where the box spans the antimeridian the
+        first value (west-most box edge) is larger than the third value
+        (east-most box edge).
+
+
+        If a feature has multiple spatial geometry properties, it is the
+        decision of the server whether only a single spatial geometry property
+        is used to determine the extent or all relevant geometries.
+      type: array
+      minItems: 4
+      maxItems: 6
+      items:
+        type: number
+      example:
+        - -110
+        - 39.5
+        - -105
+        - 40.5
+    bboxFilter:
+      type: object
+      description: Only return items that intersect the provided bounding box.
+      properties:
+        bbox:
+          $ref: '#/components/schemas/bbox'
+    timeFilter:
+      description: An object representing a time based filter.
+      type: object
+      properties:
+        time:
+          $ref: '#/components/schemas/time'
+    intersectsFilter:
+      type: object
+      description: Only returns items that intersect with the provided polygon.
+      properties:
+        intersects:
+          $ref: '#/components/schemas/polygon'
+    polygon:
+      type: object
+      properties:
+        type:
+          type: string
+          enum:
+            - polygon
+        coordinates:
+          $ref: '#/components/schemas/polygon2D'
+      required:
+        - type
+        - coordinates
+    polygon2D:
+      type: array
+      minItems: 1
+      items:
+        $ref: '#/components/schemas/linearRing2D'
+    linearRing2D:
+      type: array
+      minItems: 4
+      items:
+        $ref: '#/components/schemas/point2D'
+      example:
+        - - -104
+          - 35
+        - - -103
+          - 35
+        - - -103
+          - 34
+        - - -104
+          - 34
+        - - -104
+          - 35
+    point2D:
+      description: A 2d geojson point
+      type: array
+      minimum: 2
+      maximum: 2
+      items:
+        type: number
+        format: double
+      example:
+        - -104
+        - 35
+    time:
+      type: string
+      description: >
+        Either a date-time or a period string that adheres to RFC 3339.
+        Examples:
+
+
+        * A date-time: "2018-02-12T23:20:50Z"  
+
+        * A period:
+
+        "2018-02-12T00:00:00Z/2018-03-18T12:31:12Z" or
+
+        "2018-02-12T00:00:00Z/P1M6DT12H31M12S"
+
+
+        Only features that have a temporal property that intersects the value of
+
+        `time` are selected.
+
+
+        If a feature has multiple temporal properties, it is the decision of the
+
+        server whether only a single temporal property is used to determine the
+
+        extent or all relevant temporal properties.
+      example: '2018-02-12T00:00:00Z/2018-03-18T12:31:12Z'
+    catalogDefinition:
```

Comment:
See below, same issues as in the other catalog definition. Can we share schemas? Maybe even just reference the (hopefully updated and valid) JSON schema of the catalog?

Reply:
I would like to. I didn't really want to carry the catalogDefinition around. I'll take a stab at referencing it instead.